### ISSUE TITLE
best software

### ISSUE THREAD

**Issue by TDMidkwhatisxd** [Fukkit/Fukkit#1]
PMMP will fork this software now.

**Comment by Subject-4** [comment]
I agree, smooth af and finally a software that includes everything from singleplayer!
I must say, Fukkit is guaranteed to be your permanent server software with 24/7 fuk time and no updates, because everything is already there! 😂

**Comment by ghost** [comment]
I have a bug

**Comment by BaconOFBurger** [comment]
Its not a bug, its an undocumented feature!
